### ISSUE TITLE
Allow other content in a note overview note

### ISSUE THREAD

**Issue by therealjeybe** [JackGruber/joplin-plugin-note-overview#7]
# Current behaviour

Currently it doesn't seem to work to add other content before or after the note overview snippet or to insert a note overview into another note.
Related to this, as of now you can't use multiple note overviews in one note.

### How to reproduce
You can reproduce this by e.g. creating a note with following content and trying to recreate the note overview:

```
# All todos

<!-- note-overview-plugin
search: type:todo iscompleted:0
fields: todo_due, title, tags, notebook
sort: todo_due ASC
-->
```

# Expected / suggested behaviour

It'd be great if the note overview could also be used normally within notes, so that content before and after the snippet is possible, as well as to use multiple note overviews inside one note.

### Use cases

Personally I would like to create one note that summarizes all my ToDos from all my notebooks, categorized and labeled by certain aspects.
However, I think there could be more usecases where it could be useful to embed note overviews inside other notes. (Project index to keep track of bigger tasks, for example.)

Maybe you could share your opinion on this :-)

**Comment by JackGruber** [comment]
At the moment I work on a bigger Update to implement tow options:

1. Use of multiple note overviews in on note
2. Embed the overview in other text

Possibly the update will be available in the next two weeks.

**Comment by therealjeybe** [comment]
Great, exactly what I meant!

**Comment by JackGruber** [comment]
Options are released in v1.4.0
